### ISSUE TITLE
Blaze and Ghast projectiles

### ISSUE THREAD

**Issue by StevenB83** [necauqua/Chiseled-Me#14]
Hi! First of, great mod!

When I add this mod Blaze and Ghast projectiles seem to collide with the mob casting them.
Don't see the projectiles from Blaze, but Ghast's projectiles I can clearly see hit the Ghast casting it.
Tried on server and SP, also tried with only this mod installed. Once removed it works just fine again.
Forge versions tested: 2099, 2111, 2120, 2121, 2122, 2125 and 2151. only latest tested with only Chiseled-Me tho.
Please let me know if I can provide more information or test if you can't reproduce.

Steven.

**Comment by necauqua** [comment]
Yeah, known issue, still don't know if i ever will update this mod at all (first year of university, duh, hostel, notebook) =/
Amadornes was about to make his version(with much less ASM and a bit less bugs), but i dont know exactly.
